src/plugins/console/server/lib/utils/stream_to_string.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { IncomingMessage } from 'http';
+
+export function streamToString(stream: IncomingMessage, limit: number) {
```

Comment:
nit: maybe we can create one `streamToJSON`  method. Not sure that it will be reused in future

Reply:
addressed with e9a538909318dd0aebe1ed87248d62c7cd366f5a